dags/gmail_data_import_pipeline.py

```diff
@@ -103,19 +105,55 @@ def gmail_label_data_etl(**kwargs):
         create_table_if_not_exist(
             project_name=data_config.project_name,
             dataset_name=data_config.dataset,
-            table_name=data_config.table_name_labels,
+            table_name=table_name,
             json_schema=generated_schema
         )
-        LOGGER.info('Created table: %s', data_config.table_name_labels)
+        LOGGER.info('Created table: %s', table_name)
 
         load_file_into_bq(
             filename=filename,
             dataset_name=data_config.dataset,
-            table_name=data_config.table_name_labels,
+            table_name=table_name,
             project_name=data_config.project_name,
             auto_detect_schema=True
         )
-        LOGGER.info('Loaded table: %s', data_config.table_name_labels)
+        LOGGER.info('Loaded table: %s', table_name)
+
+
+def gmail_link_message_thread_ids_etl(**kwargs):
+    data_config = data_config_from_xcom(kwargs)
+    user_id = get_env_var_or_use_default(GMAIL_DATA_USER_ID_ENV, "")
```

Comment:
minor: perhaps now worth to extract to a separate function?

I also noticed that by this time the user id i probably required. But I am assuming that currently, if `GMAIL_DATA_USER_ID_ENV` it will default to a blank string and later fail (probably with a less descriptive error)